### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by jssol**
## Project requirements

- Watched the video with the UI example again (you have seen it while reading the Sneak Peek).
    - [UI example](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
- The console app should behaves in the same way as in the example.
- Created an `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - Listed all books.
  - Listed all people.
  - Created a person (teacher or student, not a plain `Person`).
  - Created a book.
  - Created a rental.
  - Listed all rentals for a given person's id.
- Created in `main.rb` the entry point. This method should do the following:
  - Presented the user with a list of options to perform.
  - Users can choose an option.
  - Ask for parameters for the option when needed.
  - Has a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  end
+
+  def add_rental(date, book_num, person_num)
+    book = @book_list[book_num - 1]
+    person = @people_list[person_num - 1][:value]
+    rental = Rental.new(date, book, person)
+    @rental_list.push(rental)
+  end
+
+  def display_books
```
Comment (SajjadAhmad14) [inline app.rb:37]: The user is not able to see the book list when clicking on option `1`. Kindly make sure the book list is displayed.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    student = Student.new(classroom, age, name, parent_permission)
+    @people_list.push({ value: student, type: 'Student' })
+  end
+
+  def add_teacher(specialization, age, name)
+    teacher = Teacher.new(specialization, age, name)
+    @people_list.push({ value: teacher, type: 'Teacher' })
+  end
+
+  def add_rental(date, book_num, person_num)
```
Comment (SajjadAhmad14) [inline app.rb:30]: The user is not able to create a rental when clicking on option `5`. Please make sure that the user is able to create a rental.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Jonathan, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mmsesay — APPROVED** [review]
Hi @jssol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**